dataextractor.py

```diff
@@ -0,0 +1,300 @@
+#Importing
+from rdkit import Chem, RDConfig, Geometry
```

Comment:
could import from most important to least important -- or most general to least general